modules/virt-hot-plugging-bridge-network-interface-cli.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-hot-plugging-network-interfaces.adoc              
+
+:_content-type: PROCEDURE                                             
+[id="virt-hot-plugging-bridge-network-interface_{context}"]                                   
+= Hot plugging a bridge network interface using the CLI
+
+Hot plug a bridge network interface to a virtual machine (VM) while the VM is running. 
+
+.Prerequisites
+
+* The Multus Container Network Interface (CNI) plugin is deployed on the cluster.
+* A network attachment definition is configured in the same namespace as your VM.
+* You have installed the `virtctl` tool.
+
+.Procedure
+
+. If the VM to which you want to hot plug the network interface is not running, start it by using the following command:
++
+[source,terminal]
+----
+$ virtctl start <vm_name>
+----
++
+.Example VM configuration
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: vm-fedora
+spec:
+  running: true
+  template:
+    spec:
+      domain:
+        devices:
+          interfaces:
+          - masquerade: {}
+            name: defaultnetwork
+       ...
+      networks:
+      - name: defaultnetwork
+        pod: {}
+# ...
+----
+
+. Use the following command to hot plug a new network interface to the running VM:
++
+[source,terminal]
+----
+$ virtctl addinterface <vm_name> --network-attachment-definition-name <net_attach_def_name> --name <interface_name>
+----
++
+where:
+
+<vm_name>:: Specifies the name of the `VirtualMachine` object.
+<net_attach_def_name>::  Specifies the name of `NetworkAttachmentDefinition` object.
+<interface_name>:: Specifies the name of the new network interface.
+
+
+. The `virtctl addinterface` command adds the new network interface to the VM and virtual machine instance (VMI) specification but does not attach it to the running VM. To attach the network interface to the running VM, live migrate the VM by using the following command:
```

Comment:
I might keep this sentence with the previous step and start this step with _Attach the network interface to_.